### DISCUSSION TITLE
[IntelliSense] Add hover support for Emmet-style syntax

### DISCUSSION THREAD

**Discussion by turgs** [tailwindlabs/tailwindcss#8786] [Ideas]
**What version of Tailwind CSS IntelliSense are you using?**

v0.8.4

**What version of Tailwind CSS are you using?**

3.0.23

**What package manager are you using?**

None. Using [rails/tailwindcss-rails](https://github.com/rails/tailwindcss-rails/blob/main/bin/release) that I think pull the binary directly.

I'm using Rails 7, with [propshaft](https://github.com/rails/propshaft) to deliver assets, 

**What operating system are you using?**

Ubuntu 20.04 desktop, with VS Code 1.62.2, I also have the 

**Reproduction URL**

A public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

**Describe your issue**

When hovering over the class, it doesn't show the "style-popup" that I would expect. I'm using emmet-style syntax by using Slim templates with Ruby on Rails. For example, instead of 

```
<section class="text-red-300 border border-green-300">
```

the Slim style is

```
section.text-red-300.border.border-green-300
```

I've enabled Emmet style in the settings, and that's working well with the auto-complete dropdown (once I have the [Slim](https://marketplace.visualstudio.com/items?itemName=sianglim.slim) extension also installed, but the hover isn't working.

![Peek 2022-05-22 21-32](https://user-images.githubusercontent.com/1312348/169693093-4476ad8c-d6c7-4576-8380-0ec2da702718.gif)

**Comment by bradlc** [comment]
Hey @turgs. I'm going to convert this to a discussion because it is more of a feature request than a bug report. This is maybe something we can take a look at adding in the future!
